### PR TITLE
upgrade terraform provider google beta to v5.6.0

### LINKED ISSUE #1241: Upgrade terraform-provider-google-beta to v5.1.0
Release details: https://github.com/hashicorp/terraform-provider-google-beta/releases/tag/v5.1.0

### LINKED ISSUE #1271: Upgrade terraform-provider-google-beta to v5.2.0
Release details: https://github.com/hashicorp/terraform-provider-google-beta/releases/tag/v5.2.0

### LINKED ISSUE #1280: Upgrade terraform-provider-google-beta to v5.3.0
Release details: https://github.com/hashicorp/terraform-provider-google-beta/releases/tag/v5.3.0

### LINKED ISSUE #1300: Upgrade terraform-provider-google-beta to v5.4.0
Release details: https://github.com/hashicorp/terraform-provider-google-beta/releases/tag/v5.4.0

### LINKED ISSUE #1329: V7.0.0 has a know issue with bigquery views ans schemas
### What happened?

this issue was reported in https://github.com/hashicorp/terraform-provider-google and has since been fixed (i think `5.2.0`) but pulumi `v7.0.0` uses `5.0.0`

### Example

heres links to the issues: 
https://github.com/hashicorp/terraform-provider-google/issues/16134
https://github.com/hashicorp/terraform-provider-google/issues/16192

### Output of `pulumi about`

CLI          
Version      3.92.0
Go Version   go1.21.3
Go Compiler  gc

Host     
OS       darwin
Version  14.1.1
Arch     arm64

### Additional context

_No response_

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

### LINKED ISSUE #1336: Workflow failure: Upgrade bridge
## Workflow Failure

[Upgrade bridge](https://github.com/pulumi/pulumi-gcp/blob/master/.github/workflows/upgrade-bridge.yml) has failed. See the list of failures below:

- [2023-11-14T15:34:21.000Z](https://github.com/pulumi/pulumi-gcp/actions/runs/6865840247)

- [2023-11-14T22:24:22.000Z](https://github.com/pulumi/pulumi-gcp/actions/runs/6870133543)

### PR DESCRIPTION
**Pull request by guineveresaenger**
Upgrades to upstream v5.6.0

Due to pulumi-bot being offline for a few days,there is no corresponding release issue for this version.

Closes #1300
Closes #1280
Closes #1271
Closes #1241
Closes #1329
Closes #1336

- Update patches
- Update version dependencies
- Add EdgeContainer and IntegrationConnectors modules. Map DatabaseMigrationService module properly. Add docs for new pubsubSchema resources.
- Generate schema
- make sdks

### CODE REVIEW

**Review by iwahbe — APPROVED** [review]
Looks good to me. LGTM

File: patches/0001-Allow-disabling-the-partner-name.patch [hunk 1]
```diff
@@ -1,18 +1,15 @@
-From eafcd05327894dd3f17f0ede92a78b3391b096eb Mon Sep 17 00:00:00 2001
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
```
Comment (t0yv0) [inline patches/0001-Allow-disabling-the-partner-name.patch:1]: Are these meaningful patch edits? I've found in AWS that I can avoid adding the .patch changes unless those are intentional, which makes PRs and change history much more meaningful.

**Review by t0yv0 — APPROVED** [review]

File: patches/0001-Allow-disabling-the-partner-name.patch (hunk 1 above)
Comment (guineveresaenger) [reply patches/0001-Allow-disabling-the-partner-name.patch:1]: We did have some patch conflicts in other patch files and I assumed this was a side effect of regenerating the patch set overall. 
I'm happy to discuss improvements here.

**Pull request merged**